src/component/Header/header.scss

```diff
@@ -0,0 +1,168 @@
+@import "../../assets/scss/mixin";
+#main-header {
+  @include d_flex;
+  padding: 0 0;
+  .header-logo{
+    a img {
+      width: 150px;
+    }
+  }
+  .header-logo { align-self: center}
+  .header-content-wrap {
+    display: $flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    //Start Css for Navigation
+    .header-navigation {
+      align-self: center;
+      @include transition;
+      .open-menu-btn,.close-menu-btn {
+        width: 50px;
+        height: 50px;
+        position: fixed;
+        display: none;
+        padding: 10px;
+        left: 0;
+        top: 0;
+        background: $main_color;
+        border-radius: 0 5px 5px 0;
+      }
+      //&.active {}
+      .close-menu-btn {
+        right: 0;
+        left: unset;
+        position: $position_abs;
+        padding: 13px;
+        z-index: 1;
+      }
+      & > ul {
+        @include flex_ul;
+        position: $position_rel;
+        li{
+          list-style: none;
+          padding: 10px 19px;
+          font-size: 18px;
+          position: $position_rel;
+          display: flex;
+          cursor: pointer;
+          .drop-down-arrow {
+            width: 15px;
+            margin: 0 0 0 4px;
+            @include transition;
+          }
+          &:hover .dropdownList {
+            opacity: 1;
+            visibility: visible;
+            margin-top: 0;
+          }
+          &:hover .drop-down-arrow { transform: scaleY(-1)}
+          a {
+            color: $second_color;
+            text-decoration: none;
+            display: block;
+            padding: 20px 0;
+          }
+        }
+      }
+    }
+    //End Css for Navigation
+    //Start Dropdown
+    .dropdownList {
+      position: $position_abs;
+      top: 100%;
+      left: 0;
+      width: 250px;
+      background: #444;
+      padding: 15px 20px;
+      border-radius: 5px;
+      margin: 40px 0 0;
+      box-shadow: 0 0 10px rgb(0 0 0 / 20%);
+      opacity: 0;
+      visibility: hidden;
+      @include transition;
+      &:before {
+        content: '';
+        position: absolute;
+        left: 13px;
+        top: -10px;
+        border-left: 10px solid transparent;
+        border-right: 10px solid transparent;
+        border-bottom: 10px solid #444444;
+      }
+      ul {
+        padding: 0;
+        margin: 0;
+        li {
+          padding: 5px 0;
+          font-size: 16px;
+          a {
+            padding: 5px 0;
+          }
+          a:hover {
+            opacity: 0.9;
+          }
+        }
+      }
+    }
+    //Start Dropdown
+    //Start Css fro User Image
+    .user-profile {
+      align-self: center;
+      position: $position_rel;
+      a {display: block;}

```

Comment:
we already have d-block class in reset property so no need to add again display block property.